applications/GeoMechanicsApplication/tests/cpp_tests/test_steady_state_Pw_piping_element.cpp

```diff
@@ -152,7 +152,10 @@ namespace Kratos::Testing
 
             // Create the test piping element
             std::vector<ModelPart::IndexType> element_nodes{1, 2, 3, 4};
```

Comment:
Also here, `element_nodes` appears to be no longer in use.

Reply:
Good catch, removed!